src/site/markdown/maven-artifacts.md.vm

```diff
@@ -190,10 +190,19 @@ then add the following but do not remove any Commons Logging 1.x dependencies.
 
 #dependencies(['log4j-jcl'])
 
-$h3 SLF4J Bridge
+$h3 SLF4J 2.0.x Bridge
 
-If existing components use SLF4J and you want to have this logging routed to Log4j 2, then add the
+If existing components use SLF4J 2.0.x and you want to have this logging routed to Log4j 2, then add the
 following but do not remove any SLF4J dependencies.
+Ensure that all SLF4J artifacts have consistent version.
+
+#dependencies(['log4j-slf4j2-impl'])
```

Comment:
This is the same 'add single dependency' style documentation as given below / previously for the SLF4J 1.x bridge, suggesting similar behaviour. However, the SLF4J 2.x -> Log4J 2.19 bridge currently behaves differently than prior SLF4J -> Log4j (1.x and 2.x) bridges, also requiring users add their own explicit dependency on log4j-core, because the log4j-slf4j2-impl bridge dependency doesnt actually provide it, unlike log4j-slf4j-impl.

It would be good to also fix the behaviour so its as generally expected and previously was, then this documentation would appear accurate.

Covered in https://issues.apache.org/jira/browse/LOG4J2-3601 with PR at https://github.com/apache/logging-log4j2/pull/1064